Mathlib/GroupTheory/Perm/Cycle/Type.lean

```diff
@@ -514,6 +514,15 @@ theorem _root_.exists_prime_addOrderOf_dvd_card {G : Type*} [AddGroup G] [Fintyp
 
 attribute [to_additive existing] exists_prime_orderOf_dvd_card
 
+/-- For every prime `p` dividing the order of a finite group `G` there exists an element of order
+`p` in `G`. This is known as Cauchy's theorem. -/
+@[to_additive]
+theorem _root_.exists_prime_orderOf_dvd_card' {G : Type*} [Group G] [Finite G] (p : ℕ)
```

Comment:
Should this eventually become the default? If so, then please leave a TODO on this decl and `exists_prime_orderOf_dvd_card`.

Reply:
Sure, sounds good.